lib/spack/spack/compiler.py

```diff
@@ -269,6 +270,12 @@ def check(key):
         successful.reverse()
         return dict(((v, p, s), path) for v, p, s, path in successful)
 
+    def _find_full_path(self, path):
+        return path
+
+    def setup_custom_environment(self, env):
```

Comment:
and more importantly, method docs for this -- basically just something in the superclass to say what the purpose is.